doc/modelling/binding/freundlich_ldf.rst

```diff
@@ -0,0 +1,42 @@
+.. _freundlich_ldf_model:
+
+Freundlich LDF
+~~~~~~~~~~~~~~~
+
+A Freundlich LDF isotherm model is based on the linear driving force approximation (see section :ref:`ldf_model`). It is formulated on empirical basis :cite:`Benedikt2019` :cite:`Singh2016`, and it is given as:
+
+.. math::
+	\begin{aligned} 
+		q^*_i= k_{F,i}c_{p,i}^{\frac{1}{n}}  && i = 0, \dots, N_{\text{comp}} - 1.
```

Comment:
since the exponent `n` is also component specific, shouldn't it also get an index `i`?

Reply:
You are right, the implementation is indeed considering `n` as component specific. 
It should get an index `i`.